product-service/src/functions/getProductsList/createHandler.ts

```diff
@@ -0,0 +1,22 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
+import { formatJSONResponse } from "@libs/api-gateway";
+import { Logger } from "@libs/logger/types";
+
+import { ProductRepository } from "@repositories/product/types";
+
+import schema from "./schema";
+
+export function createHandler(getProductRepository: () => ProductRepository, getLogger: () => Logger) {
+	const getProductsList: ValidatedEventAPIGatewayProxyEvent<
+		typeof schema
+	> = async (event) => {
+		try {
+			getLogger().log(event)
+			return formatJSONResponse(await getProductRepository().getAll());
+		} catch (error) {
+			return formatJSONResponse({ message: "Unknown error" }, 500);
+		}
+	};
+
+	return getProductsList;
+}
```

Comment:
identation is off in this file

Reply:
Thank you for noticing it. I have missed some files after configuration of indentation. Will fix this issue in the following tasks.